config/francestanford/config_split.config_split.francestanford.yml

```diff
@@ -1,19 +0,0 @@
-uuid: 1e0bd1f9-ce83-4b7d-9e5c-d10011665b3d
```

Comment:
This doesn't seem related, are you sure you want to remove this?

Reply:
this PR will create a new config during update hook which replaces the need for the the site specific split